packages/storybook/stories/tabs.mdx

```diff
@@ -0,0 +1,50 @@
+import { Tabs } from "@equisoft/design-elements-react";
+import { ArgTypes, Meta, Canvas, Stories } from '@storybook/blocks';
+import * as TabsStories from './tabs.stories';
+
+<Meta of={TabsStories} />
+
+# Tabs
+
+## Table of contents
+    1. [Definition](#definition)
+    2. [Usage](#usage)
+    3. [Variants](#variants)
+    4. [Properties](#properties)
+
+## Definition
+Tabs allow users to alternate between different contents that are parts of the same context [1]. 
+<Canvas of={TabsStories.Global} />
+
+## Usage
+
+### When to use
+- Use tabs to alternate between views within the same context, not to navigate to different areas. This is the single most important point because staying in place while alternating views is the reason we have tabs in the first place [1].
+- Logically chunk the content behind the tabs so users can easily predict what they'll find when they select a given tab [1].
+- Use tab only when users don't need to see content from multiple tabs simultaneously [1].
+- Use tabs that are similar in nature, and that have the same hierarchy level. If the tabs are significantly dissimilar, users will interpret them as site navigation [1].
```

Comment:
Tu veux dire quoi par "similar in nature"? est-ce que c'est le contenu ou le visuel?

Reply:
Par exemple que les tabs sont de contenu de « x » et non des contenus de différents éléments.